routes/view.js

```diff
@@ -14,10 +14,15 @@ router.get('/', health, (req, res) => {
 router.get('/cases/:date', health, defaults, async (req, res) => {
   let response = {}
   try {
-    response = await axios.get(`${req.params.apiUrl}/court/${req.params.courtCode}/cases/${req.params.date}`)
+    response = await axios.get(`${req.params.apiUrl}/court/${req.params.courtCode}/cases?date=${req.params.date}`)
   } catch (e) {
     // Silent as issue should be caught by health middleware
   }
+  if (response.data && response.data.cases) {
+    response.data.cases.forEach(item => {
+      console.log(item.data)
```

Comment:
Do we need this logging as a permanent feature? Might be a bit spammy for prod.

Reply:
That was for debugging and I mistakenly left it in. Deleted that now.